libs/ledgerjs/packages/types-live/src/account.ts

```diff
@@ -229,3 +221,16 @@ export type AccountIdParams = {
   xpubOrAddress: string;
   derivationMode: DerivationMode;
 };
+
+/**
+ * This represent the user's data part of an account which contains all user's custom information that aren't part of on-chain data
+ * The object is serializable.
+ */
+export type AccountUserData = {
```

Comment:
there is no need for `AccountUserDataRaw` because in the spec, we preferred to keep the exact same AccountRaw.

we basically have these:

**AccountRaw <-> Account + AccountUserData**